frontend/src/components/LoginForm/LoginForm.js

```diff
@@ -1,5 +1,159 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { 
```

Comment:
This doesn't seem like a PureComponent and yet you are using a functional component here. @AArlauskas pls comment

Reply:
Looks good to me. I believe we agreed that the only class components are the ones that are responsible for data fetching